donkeycar/utils.py

```diff
@@ -16,23 +16,39 @@
 import random
 import time
 import signal
+import logging
 from typing import List, Any, Tuple
 
 from PIL import Image
 import numpy as np
 
+logger = logging.getLogger()
 
 '''
 IMAGES
 '''
 ONE_BYTE_SCALE = 1.0 / 255.0
 
 
+class CString:
```

Comment:
Nit: Might want to come up with a better name ?

Reply:
Yep, done.